openapi.yaml

```diff
@@ -10613,6 +10616,294 @@ components:
         - errorText
         - errorDetail
       title: API Error
+    WhatsApp:
+      title: WhatsApp
+      x-stoplight:
+        id: 7ajh2y2qhyjk8
+      type: object
+      description: |-
+        ### Overview
+
+        Customers can send templated WhatsApp messages via Whispir's API. 
+
+        Templates must first be submitted for approval by WhatsApp in your WhatsApp Business Manager account before any template messages can be sent from Whispir. Please reach out to your Whispir account manager to help you get started.
+      x-examples:
+        WhatsApp Message Using an image:
+          to: '61430933333'
+          subject: message subject
+          whatsapp:
+            type: icebreaker
+            name: account_update_example_image_p
+            content:
+              header:
+                image:
+                  link: 'http://placekitten.com/200/300'
+              body:
+                - text: xxxxx
+                - text: yyyyy
+                - text: zzzzz
```

Comment:
Please use more meaningful examples - they're use to populate examples that developers will use to try out the API, and it's important for their developer experience to have high quality examples to enable them to get started quickly and with content that demonstrates how they might use the API.

For example:
```
to: '61430933333'
subject: Account Update
whatsapp:
  type: icebreaker
  name: account_update_example_image_p
  content:
    header:
      image:
        link: 'http://placekitten.com/200/300'
    body:
      - text: Your account has been updated.
      - text: If you did not action this update, please change your password immediately.
      - text: Thank you, Whispir
```

Reply:
I have already modified the examples and changed the title from **chore** to **feat**. Thank you